networkx/algorithms/triads.py

```diff
@@ -178,6 +178,10 @@ def triadic_census(G, nodelist=None):
     This algorithm has complexity $O(m)$ where $m$ is the number of edges in
     the graph.
 
+    For undirected graphs, the triadic census can be computed by first converting
+    the graph into a directed graph using the ``G.to_directed()`` method. Afther this conversion,
+    only the triad types 003, 102, 201 and 300 will be present in the undirected scenario.
```

Comment:
spelling check... and I made the lines shorter -- but only because of the spelling change.
```suggestion
    the graph into a directed graph using the ``G.to_directed()`` method.
    After this conversion, only the triad types 003, 102, 201 and 300 will be 
    present in the undirected scenario.
```
If you use the browser interface to commit, just be aware that you will need to `git pull` on your local version to pull that change down locally before making any other changes..